src/components/internals/calendar-body/calendar-body.styles.js

```diff
@@ -18,32 +18,30 @@ const getClearSectionStyles = props => {
     padding: ${vars.spacingXs};
     cursor: pointer;
     outline: 0;
+    transition: color ${vars.transitionStandard},
```

Comment:
We have transitions in the `input`, so we should have in the other elements as well, or else they will change colours with different timings.